daml-lf/archive/src/main/scala/com/digitalasset/daml/lf/archive/DecodeV1.scala

```diff
@@ -705,7 +705,8 @@ private[archive] class DecodeV1(minor: LanguageMinorVersion) extends Decode.OfPa
 private[lf] object DecodeV1 {
   import LanguageMinorVersion.Implicits._
 
-  private[archive] val internedIdsVersion: LanguageMinorVersion = "dev"
+  private[archive] val enumVersion: LanguageMinorVersion = "6"
```

Comment:
To accompany the `EncodeV1` suggestion.

```suggestion
  private[lf] val enumVersion: LanguageMinorVersion = "6"
```